setup.cfg

```diff
@@ -14,13 +14,13 @@ edit_on_github = False
 
 [options]
 zip_safe = False
-python_requires = >=3.6
+python_requires = >=3.8
 setup_requires =
     setuptools_scm
 
 install_requires =
     asdf >= 2.8.1
-    astropy >= 5.0.4
+    astropy >= 5.1
```

Comment:
Why this bump in a bug fix release? This just cost me half an hour trying to figure out why the ndcube figure tests were upgrading astropy suddenly.